docs/user-guide/apps/llamaedgebook.md

```diff
@@ -1,5 +1,5 @@
 ---
-sidebar_position: 7
+sidebar_position: 105
```

Comment:
Do you mean 15?

Reply:
No. I want to leave 1-100 for articles about frameworks. And 100+ for articles about applications. 

This way, I do not need to renumber all the articles if I need to add one at the top.